website/src/models/task.js

```diff
@@ -50,10 +50,14 @@ var checklist = [{
 
 var DailySchema = new Schema(
   _.defaults({
-    type: {type:String, 'default': 'daily'},
+    type: {type: String, 'default': 'daily'},
+    //TODO: Cleaner to store interval as enum instead of str?
+    frequency: {type: String, 'default': 'daily'}, // 'daily', 'weekly'
```

Comment:
Made the change https://github.com/fallenpanda1/habitrpg/commit/1af2fe33f733d000f5c6c5badabad17f67c000bf